openstudiocore/src/utilities/data/TimeSeries.cpp

```diff
@@ -626,11 +765,40 @@ namespace openstudio{
 
     std::shared_ptr<TimeSeries_Impl> TimeSeries_Impl::operator*(double d) const {
 
-      return std::shared_ptr<TimeSeries_Impl>(new TimeSeries_Impl(m_firstReportDateTime, 
-        m_secondsFromFirstReport,
-        m_values*d, 
+      return std::shared_ptr<TimeSeries_Impl>(new TimeSeries_Impl(m_startDateTime,
+        m_secondsFromStart,

```

Comment:
Sure, we can change impl interfaces, is that what you are asking?  I'm guessing we need a combination of more tests (e.g. make sure multiplying by constant doesnt change dates) and simplified impl state (so there is less chance for confusion).  Think we can do that next week?